projects/siren/index.js

```diff
@@ -3,61 +3,115 @@
   ==================================================*/
   const { request, gql } = require("graphql-request");
   const BigNumber = require('bignumber.js');
-  const sdk = require("@defillama/sdk")
+  const sdk = require("@defillama/sdk");
 
-  const GRAPH_URL = 'https://api.thegraph.com/subgraphs/name/sirenmarkets/protocol';
+  const MAINNET_GRAPH_URL = 'https://api.thegraph.com/subgraphs/name/sirenmarkets/protocol';
+
+  const POLYGON_GRAPH_URL = 'https://api.thegraph.com/subgraphs/name/sirenmarkets/protocol-v2-matic'
+
+  const POLYGON_SERIES_ADDRESS = '0x716c543b39a85aac0240ba7ed07e79f06e1fed48'
```

Comment:
First, see my comment down below about using SeriesVault, rather than SeriesController

You should fetch this from the POLYGON_GRAPH_URL, that way we can de-couple this adapter from our protocol. So, if we ever change the address used for the SeriesVault, we won't need to remember to update this address.

```
query {
  seriesVaults {
    id
  }

}

```

Reply:
Right right that makes sense ty for review ill make these edits